frontend/packages/kubevirt-plugin/src/utils/validations/vm/disk.ts

```diff
@@ -129,8 +134,13 @@ export const validateDisk = (
       addRequired(pvcName);
       validations.pvc = validatePVCName(pvcName, usedPVCNames);
     }
-  }
 
+    // TODO: implement CDROM disk bus validation
+    if (disk.getType() === DiskType.DISK) {
+      addRequired(allowedBusses.has(disk.getDiskBus()));
```

Comment:
ow sorry it should just read:
```suggestion
      addRequired(disk.getDiskBus());
```

this just checks the value is not null